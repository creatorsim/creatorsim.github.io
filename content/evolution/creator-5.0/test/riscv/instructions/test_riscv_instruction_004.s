
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    lui x5, 35
    lui x6, 22235
    lui x7, 0x34
    lui x8, 0x7F002
    lui x9, 2
