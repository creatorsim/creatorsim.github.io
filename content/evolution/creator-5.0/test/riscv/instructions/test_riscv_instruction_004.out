
cr[PC]:0x14; ir[x5,t0]:0x23000; ir[x6,t1]:0x56db000; ir[x7,t2]:0x34000; ir[x8,fp,s0]:0x7f002000; ir[x9,s1]:0x2000; keyboard[0x0]:''; display[0x0]:''; 

